settings.ui

```diff
@@ -205,6 +205,39 @@
          </layout>
         </widget>
        </item>
+       <item>
+        <widget class="QGroupBox" name="eepromGroupBox">
+         <property name="title">
+          <string>EEPROM</string>
+         </property>
+         <layout class="QGridLayout" name="eepromGridLayout">
+          <item row="0" column="0">
+           <widget class="QLabel" name="eepromImageLabel">
+            <property name="text">
+             <string>EEPROM Image:</string>
+            </property>
+           </widget>
+          </item>
+          <item row="0" column="1">
+           <widget class="QLineEdit" name="eepromPath"/>
+          </item>
+          <item row="0" column="2">
+           <widget class="QPushButton" name="setEepromPath">
```

Comment:
This name is weird.

It implies it must be pressed in order for the eepom path become active.
Something like `chooseEepromPath` or `browseEepromPath` is probably more suitable.

Reply:
Agreed, however, I was just following the previous naming scheme of the other buttons, before I was made aware that there really isn't one =)

Reply:
Changed to browseEepromPath